src/common/darktable.c

```diff
@@ -160,13 +164,13 @@ gboolean dt_supported_image(const gchar *filename)
 static void strip_semicolons_from_keymap(const char *path)
 {
   char pathtmp[PATH_MAX] = { 0 };
-  FILE *fin = fopen(path, "r");
+  FILE *fin = fopen(path, "rb");
```

Comment:
So it means that `rb` is a good solution until it will be fixed upstream, right?